<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorPickForm</class>
 <widget class="QWidget" name="ColorPickForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>选择皮肤</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../rs/rs.qrc">
    <normaloff>:/sys/qlogo.png</normaloff>:/sys/qlogo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font: 75 9pt &quot;微软雅黑&quot;;
}

#color_main_widget{
	border-image: url(:/sys/back.png) 5;
	border-radius:2px;
}

#centralwidget{
	border-radius:2px;
	background-color: rgb(22, 154, 218);
}

/*标题栏*/
#color_main_top{
	
	border-top-left-radius:2px;
	border-top-right-radius:2px;
}

#color_main_top *{
	color:#fff;
	font-size:10pt;
}

#color_main_con{
	
	background-color: rgb(255, 255, 255);
	border-bottom-left-radius:2px;
	border-bottom-right-radius:2px;
}

#color_main_con #widget_tool{
	background-color: rgb(237, 238, 238);
	border-bottom:1px solid rgb(220,220,220);
}

#color_main_con #widget_colors{

}

#color_main_con #color_items QPushButton,#color_panle_pushButton{
	background-color: rgb(22, 154, 218);
	border:none;
}

#color_main_con #color_items QPushButton:hover,
#color_panle_pushButton:hover
{
	border:3px solid rgb(39,140,222);
}

#color_panle_pushButton{
	background-image: url(:/sys/color_panle.png);
	
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="color_main_widget" native="true">
     <property name="minimumSize">
      <size>
       <width>752</width>
       <height>300</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="centralwidget" native="true">
        <property name="styleSheet">
         <string notr="true">#centralwidget{
 
	
}</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="1" column="0">
          <widget class="QWidget" name="color_main_con" native="true">
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="widget_colors" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0">
               <property name="spacing">
                <number>1</number>
               </property>
               <property name="leftMargin">
                <number>6</number>
               </property>
               <property name="topMargin">
                <number>6</number>
               </property>
               <property name="rightMargin">
                <number>6</number>
               </property>
               <property name="bottomMargin">
                <number>6</number>
               </property>
               <item>
                <widget class="QWidget" name="widget_2" native="true">
                 <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1">
                  <property name="spacing">
                   <number>1</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QWidget" name="color_items" native="true">
                    <property name="minimumSize">
                     <size>
                      <width>100</width>
                      <height>40</height>
                     </size>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_2">
                     <property name="spacing">
                      <number>1</number>
                     </property>
                     <property name="leftMargin">
                      <number>0</number>
                     </property>
                     <property name="topMargin">
                      <number>0</number>
                     </property>
                     <property name="rightMargin">
                      <number>0</number>
                     </property>
                     <property name="bottomMargin">
                      <number>0</number>
                     </property>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="color_panle_pushButton">
                    <property name="minimumSize">
                     <size>
                      <width>40</width>
                      <height>40</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>40</width>
                      <height>40</height>
                     </size>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>0</width>
                      <height>0</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QWidget" name="color_main_top" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>39</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>6</number>
            </property>
            <property name="topMargin">
             <number>6</number>
            </property>
            <property name="rightMargin">
             <number>6</number>
            </property>
            <property name="bottomMargin">
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../../rs/rs.qrc">:/sys/qlogo.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>更改外观</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="styleSheet">
               <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
              </property>
              <property name="text">
               <string>X</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../rs/rs.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ColorPickForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>716</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>835</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
